sdks/java/core/src/main/java/org/apache/beam/sdk/testing/StaticWindows.java

```diff
@@ -100,9 +102,22 @@ public boolean isCompatible(WindowFn<?, ?> other) {
   }
 
   @Override
-  public BoundedWindow getSideInputWindow(BoundedWindow window) {
-    checkArgument(windows.get().contains(window),
-        "StaticWindows only supports side input windows for main input windows that it contains");
-    return window;
+  public WindowMappingFn<BoundedWindow> getDefaultWindowMappingFn() {
```

Comment:
It's a package-private implementation detail of `PAssert`; I don't think should be parameterized, and we can still change it if we desire.